physical/consul.go

```diff
@@ -513,7 +513,13 @@ shutdown:
 // checkID returns the ID used for a Consul Check.  Assume at least a read
 // lock is held.
 func (c *ConsulBackend) checkID() string {
-	return "vault-sealed-check"
+	return fmt.Sprintf("vault-sealed-check:%s:%d", c.advertiseHost, c.advertisePort)
```

Comment:
This might be better done as `fmt.Sprintf("%s:vault-sealed-check", c.serviceID())` so we always uniquify in the same way.